Split/Network/Streaming/DefaultSseNotificationParser.swift

```diff
@@ -8,7 +8,23 @@
 
 import Foundation
 
-class NotificationParser {
+protocol SseNotificationParser {
+
+    func parseIncoming(jsonString: String) -> IncomingNotification?
+
+    func  parseSplitUpdate(jsonString: String) throws -> SplitsUpdateNotification
```

Comment:
a formatting detail: use single space instead of double space (i.e. `func  parse...` --> `func parse...`). the same detail is in other methods of the protocol.

Reply:
Fixed!